sdf_timing/tests/test_parse_and_write.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env python3
+# coding: utf-8
+
+# Copyright 2022 Tomas Brabec
```

Comment:
can we stay with F4PGA Authors (as it is in all the other files)?

Reply:
I guess so as long as you eventually merge in the pull request; there is nothing special about the test code that I would regret losing copyright to. You may then add me to `AUTHORS` if you like and consider this pull request a significant contribution; that's really up to you.

So once we resolve all the other comments, I will change the notice.